pyproject.toml

```diff
@@ -0,0 +1,130 @@
+authors = ["TACC-WMA <wma-portals@tacc.utexas.edu>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+aldryn-apphooks-config = "0.6.0"
+aldryn-boilerplates = "0.8.0"
+aldryn-common = "1.0.5"
+aldryn-search = "1.1.0"
+asgiref = "3.2.10"
+astroid = "2.4.2"
+autopep8 = "1.5.4"
+certifi = "2020.6.20"
+chardet = "3.0.4"
+colorclass = "2.2.0"
+Django = "2.2.16"
+django-appconf = "1.0.4"
+django-appdata = "0.3.0"
+django-auth-ldap = "2.2.0"
+django-classy-tags = "2.0.0"
+django-cms = "3.7.4"
+django-filer = "2.0.2"
+django-formtools = "2.2"
+django-haystack = "3.0"
+django-ipware = "3.0.1"
+django-js-asset = "1.2.2"
+django-meta = "1.7.0"
+django-mptt = "0.11.0"
+django-parler = "2.2"
+django-polymorphic = "3.0.0"
+django-sekizai = "2.0.0"
+django-settings-export = "1.2.1"
+django-sortedm2m = "3.0.2"
+django-spurl = "0.6.7"
+django-standard-form = "1.1.1"
+django-taggit = "1.2.0"
+django-taggit-autosuggest = "0.3.8"
+django-taggit-templatetags = "0.2.5"
+django-templatetag-sugar = "1.0"
+django-treebeard = "4.3.1"
+djangocms-admin-style = "2.0.0"
+djangocms-apphook-setup = "0.4.1"
+djangocms-attributes-field = "2.0.0"
+djangocms-blog = "1.1.1"
+djangocms-bootstrap4 = "2.0.0"
+djangocms-column = "1.11.0"
+djangocms-file = "3.0.0"
+djangocms-forms = "0.2.5"
+djangocms-googlemap = "2.0.0"
+djangocms-icon = "2.0.0"
+djangocms-link = "3.0.0"
+djangocms-page-meta = "0.10.0"
+djangocms-picture = "3.0.0"
+djangocms-snippet = "3.0.0"
+djangocms-style = "3.0.0"
+djangocms-text-ckeditor = "3.10.0"
+djangocms-transfer = "1.0.0"
+djangocms-video = "3.0.0"
+django-test-without-migrations = "0.6"
+docopt = "0.6.2"
+easy-thumbnails = "2.7"
+elasticsearch = "7.9.1"
+elasticsearch-dsl = "7.3.0"
+flake8 = "3.8.3"
+gunicorn = "20.0.4"
+hashids = "1.3.1"
+html5lib = "1.1"
+idna = "2.10"
+importlib-metadata = "1.7.0"
+isort = "5.5.2"
+jedi = "0.17.2"
+jsonfield = "3.1.0"
+lazy-object-proxy = "1.4.3"
+lxml = "4.5.2"
+mccabe = "0.6.1"
+packaging = "20.4"
+parso = "0.7.1"
+Pillow = "9.0.0"
+pip-upgrader = "1.4.15"
+pluggy = "0.13.1"
+psycopg2 = "2.8.6"
+psycopg2-binary = "2.8.6"
+pyasn1 = "0.4.8"
+pyasn1-modules = "0.2.8"
+pycodestyle = "2.6.0"
+pydocstyle = "5.1.1"
+pyflakes = "2.2.0"
+pylint = "2.6.0"
+pyparsing = "2.4.7"
+python-dateutil = "2.8.1"
+python-jsonrpc-server = "0.4.0"
+python-language-server = "0.35.0"
+python-ldap = "3.3.1"
+pytz = "2020.1"
+requests = "2.24.0"
+rope = "0.17.0"
+six = "1.15.0"
+snowballstemmer = "2.0.0"
+sqlparse = "0.3.1"
+tablib = "2.0.0"
+terminaltables = "3.1.0"
+toml = "0.10.1"
+typed-ast = "1.4.1"
+ujson = "3.2.0"
+Unidecode = "1.1.1"
+urllib3 = "1.25.10"
+URLObject = "2.4.3"
+uWSGI = "2.0.19.1"
+webencodings = "0.5.1"
+wrapt = "1.12.1"
+yapf = "0.30.0"
+YURL = "1.0.0"
+zipp = "3.1.0"
+
+[tool.poetry.dev-dependencies]
+mock = "^4.0.2"
+pytest-mock = "^3.1.1"
+pytest = "^5.4.3"
+pytest-cov = "^2.9.0"
+pytest-django = "^3.9.0"
+flake8 = "^3.8.2"
+coverage = "^5.1"
+requests-mock = "^1.8.0"
```

Comment:
These look new, and copied from Portal.

Are you preparing for a future in which we use CMS test cases?